src/coreclr/jit/emitarm64.cpp

```diff
@@ -5875,6 +5875,9 @@ void emitter::emitIns_R_R_I(instruction     ins,
                 return;
             }
 
+            // reg2 can be alread encoded to zero
```

Comment:
```suggestion
            // reg2 can be already encoded to zero
```

Reply:
ok to fix in next batch? don't want to run CI again.

Reply:
Sure